res/xml/qwertz.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<keyboard>
+  <row>
+    <key key0="q" key1="esc" key2="1" key3="\@" key4="!"/>
+    <key key0="w" key1="^" key2="2" key3="&quot;" key4="°"/>
```

Comment:
`°` is already `(accent)^+*`, which is only available for french. What do you think of adding `Fn+*` as a combination for `°` ?

Reply:
Maybe? I don't mind too much. Just thought it was practical, as it's easily reachable on a German keyboard by just pressing Shift and then the top left key. It's drawn on the key.

Reply:
I've changed the combination to `Fn+*`. No need to dedicate a key now.

Reply:
> It's drawn on the key.

That's a limitation of this app. There can be up to 5 characters drawn on a key already, it's not possible to draw all the combinations: shift, fn and then all the accents.